buildSrc/src/main/groovy/org/elasticsearch/gradle/test/StandaloneTestPlugin.groovy

```diff
@@ -46,5 +46,10 @@ public class StandaloneTestPlugin implements Plugin<Project> {
         test.testClassesDir project.sourceSets.test.output.classesDir
         test.mustRunAfter(project.precommit)
         project.check.dependsOn(test)
+
+        project.namingConventionsCheck {
+            // Stand alone tests don't follow the rules!
+            skipIntegTestInDisguise = true
```

Comment:
OK - regardless of what you say I'm going to remove this and try and fix those names/apply this param closer to the offending test.